src/field_theory/adjoin.lean

```diff
@@ -534,4 +534,45 @@ induction_on_adjoin_fg P base ih K K.fg_of_noetherian
 
 end induction
 
+section alg_hom_mk_adjoin_splits
+
+variables {F E K : Type*} [field F] [field E] [field K] [algebra F E] [algebra F K] {S : finset E}
+
+lemma alg_hom_mk_adjoin_splits
+(hK : ∀ x ∈ S, ∃ H : is_integral F (x : E), (minimal_polynomial H).splits (algebra_map F K)) :
+nonempty ((adjoin F (S : set E)) →ₐ[F] K) :=
```

Comment:
```suggestion
  (hK : ∀ x ∈ S, ∃ H : is_integral F (x : E), (minimal_polynomial H).splits (algebra_map F K)) :
  nonempty ((adjoin F (S : set E)) →ₐ[F] K) :=
```

Reply:
The assumption looks a bit ugly... I'll have to think about how we can improve that.